pyemma/msm/estimators/maximum_likelihood_msm.py

```diff
@@ -897,7 +897,7 @@ class OOMReweightedMSM(_MSMEstimator):
     r"""OOM based estimator for MSMs given discrete trajectory statistics"""
 
     def __init__(self, lag=1, reversible=True, count_mode='sliding', sparse=False, connectivity='largest',
-                 dt_traj='1 step', nbs=10000, tol_rank=10.0):
+                 dt_traj='1 step', nbs=10000, trajectory_bootstrap=True, tol_rank=10.0):
```

Comment:
How about this:
``rank_C0 = 'bootstrap_counts'``
where ``bootstrap_counts`` and ``bootstrap_trajs`` are options. This allows us to choose other rank selection methods later. I'd choose bootstrap_counts as default as it always works, while bootstrap_trajs is only meaningful if you have enough trajectories.